WebDriverAgentTests/IntegrationTests/FBW3CTouchActionsIntegrationTests.m

```diff
@@ -295,6 +295,13 @@ - (void)testTap
 
 - (void)testDoubleTap
 {
+  if ([UIDevice.currentDevice.systemName  isEqual: @"iPadOS"] && SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"15.0")) {
```

Comment:
Cannot we use [interface idiom](https://stackoverflow.com/questions/10167221/ios-detect-if-user-is-on-an-ipad) instead to detect ipad?